### PR TITLE
Added minutes for meeting on 2017-02-15

### PR DESCRIPTION
**Pull request by sedders123**

### CODE REVIEW

File: 2017/2017-02-15.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Members split into smaller groups and discussed what they wanted from the society.
+
+Ideas put forward include:
+- A coding club - A regular meeting in an IT room where people of all skills can come to learn, teach or work on group projects. Both hardware and software projects.
+- Attending Hackathons and possibly organising our own for Chi students or feeder colleges
+-  Building (or purchasing) a robotic arm, and program it to solve problems
+- Organising social events - Arrange get togethers to socialise and relax, outside of a working environment.
+
+### Actions
+JH, AS and JSS to devise plan to get code club set up
```
Comment (adamsandle) [inline 2017/2017-02-15.md]: AS/JSS to setup github student

Comment (sedders123) [inline 2017/2017-02-15.md]: Yep forgot about that, will add now

**Comment by sedders123** [comment]
I put the action under Election + Society Admin

File: 2017/2017-02-15.md (hunk 1 above)
Comment (sedders123) [inline 2017/2017-02-15.md]: I put the action under Election + Society Admin

**Pull request merged**